tests/thermodynamics/test_hessianJob.py

```diff
@@ -49,5 +49,33 @@ def test_forces(self):
         self.assertAlmostEqual(self.job.output.pressures[0, 0, 0], -0.03)
 
 
+class TestSelectiveDynamics(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.file_location = os.path.dirname(os.path.abspath(__file__))
+        cls.project = Project(
+            os.path.join(cls.file_location, "selective_dynamics")
+        )
+        cls.project.remove_jobs_silently(recursive=True)
+
+    def test_selective_dynamics(self):
+        structure = Atoms(
+            positions=[[0, 0, 0], [1, 1, 1]], elements=["Fe", "Fe"], cell=2 * np.eye(3)
+        )
+        structure.add_tag(selective_dynamics=None)
+        structure.selective_dynamics[0] = [True, True, True]
+        structure.selective_dynamics[1] = [False, False, False]
+        job = self.project.create_job("HessianJob", "hess")
+        job.structure = structure
+        job.to_hdf()
+        job_reload = self.project.create_job("HessianJob", "hess")
+        job_reload.from_hdf()
+        structure_reload = job_reload.structure
+        structure_reload.add_tag(selective_dynamics=None)
```

Comment:
I thought the default value should be something like `[True, True, True]` or `[False, False, False]`?

Reply:
Typically I use `None` to apply multiple filters and afterwards check which atoms have not been specified yet.